testsuite/classlibrary/TestPprotect.sc

```diff
@@ -1,6 +1,7 @@
 
 TestPprotect : UnitTest {
-	test_Pprotect_resetExceptionHandlerOnError {
+
+	test_reset_exceptionHandler_on_error {
```

Comment:
why? the previous name was actually better

Reply:
The `Pprotect` is redundant, but I recovered the rest.